tox.ini

```diff
@@ -9,9 +9,9 @@ deps =
 commands =
     pip install -e .[d]
     coverage erase
-    coverage run -m pytest tests -m "not python2" {posargs}
+    pytest tests -m "not python2" --numprocesses auto --cov src/ --cov tests/ {posargs}
```

Comment:
pytest does not respect `.coveragerc` ?

Reply:
I'm not actually sure, but good point, I'll check again.

Reply:
Okay so `pytest-cov` does respect `.coveragerc`. I just assumed that `--cov` needed an argument so coverage would be collected but I was wrong. `pytest tests -m "not python2" --numprocesses auto --cov {posargs}` works just fine. Great catch! Thank you!